src/components/layouts/EmptyState/DashboardEmptyState.tsx

```diff
@@ -0,0 +1,50 @@
+import { PrimaryBtn } from '@/components/buttons/PrimaryBtn'
+import { AppMargin, SizeofAppMargin } from '@/hoc/AppMargin'
+import { AddIcon, TasksListIcon } from '@/icons'
+import { setShowModal } from '@/redux/features/createTaskSlice'
+import store from '@/redux/store'
+import { UserType } from '@/types/interfaces'
+import { SxCenter } from '@/utils/mui'
+import { Box, Stack, Typography } from '@mui/material'
+
+const DashboardEmptyState = ({ userType }: { userType: UserType }) => {
+  return (
+    <AppMargin size={SizeofAppMargin.LARGE} py="20px">
+      <Box
+        sx={{
+          display: 'flex',
+          height: '80vh',
+          ...SxCenter,
+        }}
+      >
+        <Stack rowGap={'20px'} direction={'column'} sx={{ width: '453px' }}>
+          <Stack rowGap={'12px'} direction={'column'}>
+            <TasksListIcon />
```

Comment:
This icon has design inconsistency. 

In this PR - 
![Screenshot from 2024-07-11 09-29-21](https://github.com/copilot-platforms/tasks-app/assets/38468429/1fd4d7fb-2442-468c-98a8-a60345ef0ffd)

In Figma - 
![Screenshot from 2024-07-11 09-30-30](https://github.com/copilot-platforms/tasks-app/assets/38468429/4cb5127a-0995-4441-826d-107fc4729475)

We are missing the background and border radius for the background.